test/tests.jl

```diff
@@ -55,6 +56,7 @@ PkgAuthentication.register_open_browser_hook(url -> HTTP.get(url))
     @test success2 isa PkgAuthentication.Success
     @test success2.token["expires_at"] > time()
     @test success2.token["refresh_token"] !== success.token["refresh_token"]
+    @test startswith(success2.token["id_token"], "refresh-")
```

Comment:
I am prepending different prefixes to the tokens now, depending on whether it comes via the refresh route, or the full auth route. This is to make sure that `success2` does actually come from the `NeedRefresh` state, and hasn't fallen back to the full authentication.